### ISSUE TITLE
Invalid value for input "iam_role_additional_policies"

### ISSUE THREAD

**Issue by tanvp112** [terraform-aws-modules/terraform-aws-eks#2136]
Hi,

The [fargate_profile example](https://github.com/terraform-aws-modules/terraform-aws-eks/tree/master/examples/fargate_profile) has broken with release 18.26.2; the variable [iam_role_additional_policies](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/661db03fb5aab04bc72916e60da1ac63bc6bf91a/modules/fargate-profile/variables.tf#L71) was declared as list of strings, but the fargate module wants to [assign map](https://github.com/terraform-aws-modules/terraform-aws-eks/blob/661db03fb5aab04bc72916e60da1ac63bc6bf91a/node_groups.tf#L221) to it.

```
 Error: Invalid value for input variable
│
│   on ..\..\node_groups.tf line 221, in module "fargate_profile":
│  221:   iam_role_additional_policies  = try(each.value.iam_role_additional_policies, var.fargate_profile_defaults.iam_role_additional_policies, {})
│
│ The given value is not suitable for
│ module.eks.module.fargate_profile["secondary"].var.iam_role_additional_policies
│ declared at ..\..\modules\fargate-profile\variables.tf:71,1-40: list of
│ string required.
╵
╷
│ Error: Invalid value for input variable
│
│   on ..\..\node_groups.tf line 221, in module "fargate_profile":
│  221:   iam_role_additional_policies  = try(each.value.iam_role_additional_policies, var.fargate_profile_defaults.iam_role_additional_policies, {})
│
│ The given value is not suitable for
│ module.eks.module.fargate_profile["default"].var.iam_role_additional_policies
│ declared at ..\..\modules\fargate-profile\variables.tf:71,1-40: list of
│ string required.
```

**Comment by kuboraam** [comment]
+1. was just about to open an issue for the same. Glad I found this thread first. Works fine in 18.26.1.

**Comment by tehseensajjad** [comment]
+1
Changing line 221 in `/.terraform/modules//cluster.eks/node_groups.tf` this the following removes the error
```
  iam_role_additional_policies  = try(each.value.iam_role_additional_policies, var.fargate_profile_defaults.iam_role_additional_policies, [])
```

**Comment by antonbabenko** [comment]
This issue has been resolved in version 18.26.3 :tada:
